server/server.js

```diff
@@ -5,22 +5,22 @@ const path = require("path");
 const bodyParser = require("body-parser");
 const env = require("dotenv").config();
 const PORT = process.env.PORT;
+const queriesRouter = require("./queriesRoutedToDB");
 
 app.use(cors());
 
-//Start writing all the middleware below
-
 /**
  * handle parsing request body
  */
 app.use(express.json());
 
-//handle page not found
-app.use((req, res) =>
-  res.status(404).send("This is not the page you're looking for...")
-);
+//handle workouts-list query for workout cards data
+app.get("/workouts-list", queriesRouter.getWorkoutsList, (req, res) => {
```

Comment:
workouts-list route does not match workoutList proxy

Reply:
fixed